x/distribution/keeper/delegation.go

```diff
@@ -9,13 +9,15 @@ import (
 // initialize starting info for a new delegation
 func (k Keeper) initializeDelegation(ctx sdk.Context, val sdk.ValAddress, del sdk.AccAddress) {
 	// period has already been incremented
```

Comment:
Can this comment or variable name be changed to indicate that the period being dealt with here is the period ended by this delegation action?

Reply:
Clarified comment.